llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/utils.py

```diff
@@ -107,18 +109,52 @@ def get_text_from_response(self, response: dict) -> str:
 
 
 def completion_to_anthopic_prompt(completion: str) -> str:
-    return messages_to_anthropic_prompt(prompt_to_messages(completion))
+    return messages_to_anthropic_messages(prompt_to_messages(completion))
```

Comment:
```suggestion
    messages, _ = messages_to_anthropic_messages(prompt_to_messages(completion))
    return messages
```

`messages_to_anthropic_messages` in the anthropic utils have different signature. It returns a tuple with messages and system prompt.

Reply:
This will drop the system prompt though? I though I added a `_messages_to_anthropic_messages()` function to handle this?